### DISCUSSION TITLE
SwiftUI - How to modify the text displayed on the screen

### DISCUSSION THREAD

**Discussion by pedrohgmuniz** [community/community#36462] [Mobile] [answered]
Hello! I'm starting with iOS development and SwiftUI (practically a complete noob), so can anyone give me some pointers or anything that could help me edit text on the screen (put it anywhere I want, change color, font, size, style, etc. etc.)?

**Comment by ameninadogorro** [comment, marked as answer]
I was looking for that recently and found a guide (https://www.appcoda.com/learnswiftui/swiftui-text.html) that helped me get going with text in SwiftUI. But I also did some trial and error on Xcode and ended up finding what I wanted (I guess).
I wanted to display the first text seen on my app at the top, centered, in a relatively big font (so as to look like a title), and bold. 
This is how that piece of my code looks like:
```
Text("The piece of text you want to display")
                    .fontWeight(.bold)
                    .font(.largeTitle)
                    .padding().frame(width: 400, height: 150, alignment: .center)

```
From the guide I found I simply saw that I could use stuff like .fontWeight or .font, and then I just started typing the code on Xcode and experimented with different options and parameters.
That .frame thingy I found on my own, while exploring Xcode like that (by typing the dot and testing the options I found).

Afterwards I found out that these elements which come after a dot - such as .padding() - are called "modifiers", and they modify a given view in order of appearance: that is, sort of like an onion, in which the exterior layers modify the ones closer to the center. Depending on the kind of modifier you're using, the order in which you write them may change the end result. In my case (piece of code above), the order of the modifiers didn't really matter. So my tip is: experiment!
